CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ The following is a curated list of changes in the Enact eslint config:
 
 ## unreleased
 
+* Added `browser`, `node`, and `jest` environment.
```

Comment:
If we want to release including `worker` and the new environments altogether, changelogs need to be merged in a single line.

Reply:
Agree